core/include/traccc/seeding/track_params_estimation_helper.hpp

```diff
@@ -83,12 +83,13 @@ inline TRACCC_HOST_DEVICE bound_vector seed_to_bound_vector(
 
     // Curvature (with a sign) estimate
     scalar rho = -2.0f * B / getter::perp(vector2{1., A});
+
     // The projection of the top space point on the transverse plane of
     // the new frame
     scalar rn = local2[0] * local2[0] + local2[1] * local2[1];
     // The (1/tanTheta) of momentum in the new frame,
     scalar invTanTheta =
-        local2[2] * std::sqrt(1.f / rn) / (1.f + rho * rho * rn);
+        local2[2] * std::sqrt(1.f / rn) / (1.f + rho * rho * rn / 24.);
```

Comment:
Think I spotted a bug - without this term track_parameter_estimation test fails :p